ElementX/Sources/Screens/Settings/NotificationSettingsEditScreen/NotificationSettingsEditScreenModels.swift

```diff
@@ -33,11 +33,20 @@ struct NotificationSettingsEditScreenViewState: BindableState {
     var defaultMode: NotificationSettingsEditScreenDefaultMode?
     var pendingMode: NotificationSettingsEditScreenDefaultMode?
     var roomsWithUserDefinedMode: [NotificationSettingsEditScreenRoom] = []
+    var canHomeServerPushEncryptedEvents = false
```

Comment:
```suggestion
    var canHomeserverPushEncryptedEvents = false
```
because it's called `xxxHomeserverXxx` everywhere else